configure.ac

```diff
@@ -136,6 +136,15 @@ AC_ARG_ENABLE([library],
 
 AM_CONDITIONAL([WITH_LIBRARY], [test "x$with_library" = "xyes"])
 
+# Add configure option to disable documentation building
+AC_ARG_ENABLE([doc],
+    [AS_HELP_STRING([--disable-doc],
```

Comment:
Do you want me to change "--disable-doc" to "--enable-doc" ? Should I leave docs build enabled by default?

Reply:
My comment was driven by the similar case we handled recently ( you need to click "open resolved" for configure.ac):
https://github.com/intel/ledmon/pull/115#pullrequestreview-1516785361

I thought that it is similar case but now I can see that the default behavior is reversed, so it cannot be `--enable-doc`.
It must be `--disable-doc` because we would like default it to no if not set.

Please see that you make flag `--disable-doc` but you are printing `--enable-doc=yes/no` in "configure parameters" 
sections. It is confusing because as a user I would like to copy the configure parameters, I'm not sure if it will work now.

Please then update  "configure parameters" to match the field and it should be fine.
Thanks!

Reply:
>  It is confusing because as a user I would like to copy the configure parameters, I'm not sure if it will work now.
Even if the user will copy `--enable-doc=yes/no` it will work as expected.
Here is what autoconf doc says (chapter 15.3 Choosing Package Options):
```
The options have one of these forms:

     --enable-feature[=arg]
     --disable-feature
....

--disable-feature is equivalent to --enable-feature=no. 

```

So I tested all four cases:
1. `--disable-doc` (docs not created)
2. `--enable-doc=no` (docs not created)
3. `--enable-doc=yes` (docs created)
4. without this option (docs created)

I'm agree with you that it may be a little bit confusing for user when `--disable-doc` was specified, but in result `configure parameters`  will contains `--enable-doc=no`. 

But I can't change this behavior because autoconf can't handle arguments(yes/no) for `--disable` :shrug: